package.json

```diff
@@ -4368,6 +4369,7 @@
         "i18n-ts": "^1.0.5",
         "immutable": "^4.3.0",
         "js-yaml": "^4.1.0",
+        "jsdom": "^23.0.1",
```

Comment:
Interesting, jsdom is mentioned at https://github.com/aws/aws-toolkit-vscode/blob/4f9e41a8a13195d04de98df34af47ee8d794dea4/docs/ARCHITECTURE.md?plain=1#L212

Reply:
can this be a devDependency? doesn't look like it's needed in production.